bots/task/github.py

```diff
@@ -81,11 +83,29 @@ def write(self, value):
         with open(self.path, 'a') as f:
             f.write(value)
 
+def get_origin_repo():
+    res = subprocess.run([ "git", "remote", "get-url", "origin" ], capture_output=True)
+    if res.returncode != 0:
+        # Not a git repo
+        return None
+    url = res.stdout.decode('utf-8').strip()
+    m = re.match("^git@github.com:(.*)", url)
+    if m:
+        return m.group(1)
+    m = re.match("^https://github.com/(.*)", url)
+    if m:
+        return m.group(1)
+    raise RuntimeError("Not a GitHub repo: %s" % url)
+
 class GitHub(object):
     def __init__(self, base=None, cacher=None, repo=None):
         if base is None:
             if repo is None:
-                repo = os.environ.get("GITHUB_BASE", "cockpit-project/cockpit")
+                repo = os.environ.get("GITHUB_BASE", None)
+            if repo is None:
+                repo = get_origin_repo()
+            if repo is None:
+                repo = "cockpit-project/cockpit"
```

Comment:
Good question.  I kept it in just in case.  Maybe people are running tests-trigger while the are not in any git repo.

But, yes, having a silent default might be confusing.  I am happy to make this case an error.